sys/hashes/hashes.c

```diff
@@ -109,3 +110,26 @@ uint32_t one_at_a_time_hash(const uint8_t *buf, size_t len)
     hash += hash << 15;
     return hash;
 }
+
+uint8_t crc8(const uint8_t *data, size_t len)
```

Comment:
This should go to a separate file `crc8.c` in `sys/checksum`, see the commend for the header file.